packages/site/src/pages/index.tsx

```diff
@@ -36,148 +51,418 @@ const features = [
+            gridColumn: '2 / span 10',
+            textAlign: 'center',
+            [mediaQuery.medium]: {
+              alignItems: 'start',
+              gridColumn: '1 / span 6',
+              textAlign: 'left',
+            },
+          }}
+        >
+          <Text variant="heading1">
+            Build Your Own <br /> <Text variant="mark">Design Tool</Text>
+          </Text>
+          <Spacer size="2rem" />
+          <Text variant="body1" width="32ch">
+            Noya is an ecosystem of design utilities to help with your next
+            project.
+          </Text>
+          <Spacer size="4rem" />
+          <Button>Open App</Button>
+        </Stack>
+
+        <Stack
+          alignItems="center"
+          justifyContent="center"
+          minHeight="24rem"
+          background="rgb(141, 47, 216)"
+          css={{
+            gridColumn: '1 / -1',
+            [mediaQuery.medium]: {
+              gridColumn: '7 / span 6',
+              minHeight: '40rem',
+            },
+          }}
+        >
+          Demo Here
+        </Stack>
+      </Grid>
+
+      <Spacer size="24rem" />
+
+      <Grid
+        gridColumn="1 / -1"
+        gridTemplateColumns="repeat(12, 1fr)"
+        gap="1rem"
+        css={{
+          rowGap: '6rem',
+          [mediaQuery.medium]: {
+            rowGap: '1rem',
+          },
+        }}
+      >
+        <Grid
+          gridColumn="1 / -1"
+          gridTemplateColumns="repeat(2, 1fr)"
+          alignItems="end"
+          css={{
+            gridRow: '2',
+            gap: '4rem',
+            [mediaQuery.medium]: {
+              gridRow: '1',
+              gridColumn: '1 / span 6',
+              gap: '1rem',
+            },
+          }}
+        >
+          {features.map((feature) => (
+            <Stack
+              key={feature.name}
+              alignItems="center"
+              css={{
+                [mediaQuery.medium]: {
+                  flexDirection: 'row',
+                },
+              }}
+            >
+              <Text variant="body1">{feature.name}</Text>
+            </Stack>
+          ))}
+        </Grid>
+
+        <Stack
+          gap="2rem"
+          gridColumn="2 / span 10"
+          css={{
+            gridRow: '1',
+            textAlign: 'center',
+            [mediaQuery.medium]: {
+              gridColumn: '7 / span 6',
+              textAlign: 'left',
+            },
+          }}
+        >
+          <Text variant="heading2">
+            All Of The Features <br /> You’re Used To <br />
+            <Text variant="mark">{preventTextRunt('And More')}</Text>
+          </Text>
+          <Text variant="body1">
+            Full-featured while delivering a system that can cater to your
+            specific needs.
+          </Text>
         </Stack>
 
-        <Spacer.Vertical size="12rem" />
+        <Spacer
+          size="6rem"
+          css={{
+            [mediaQuery.medium]: {
+              minHeight: '24rem',
```

Comment:
Not happy with this and can cleanup later, instead of `size` maybe just `minHeight` for now. I was only using `Spacer` for vertical alignment since columns should be used for horizontal alignment.